src/backend/app-core/repository/tokens/pgsql_tokens.go

```diff
@@ -161,7 +161,7 @@ func (p *PgsqlTokenRepository) FindAuthToken(userGUID string, encryptionKey []by
 	err := p.db.QueryRow(findAuthToken, userGUID).Scan(&ciphertextAuthToken, &ciphertextRefreshToken, &tokenExpiry)
 	if err != nil {
 		msg := "Unable to Find UAA token: %v"
-		log.Debugf(msg, err)
+		log.Errorf(msg, err)
```

Comment:
I think this needs to stay Debugf, otherwise verifySession without a valid session logs this every time
- the error handling in this case should be handled a level up in the code.